src/common/paths/query_ast.yml

```
@@ -1,9 +1,34 @@
 post:
   operationId: PostQueryAst
-  description: Analyzes flux query and generates a query specification.
   tags:
     - Query
-  summary: Generate an Abstract Syntax Tree (AST) from a query
+  summary: Generate an Abstract Syntax Tree (AST) from a Flux query
+  description: |
+    Analyzes a Flux query and generates a complete package source AST from the
+    query.
+
+    ASTs are tree representations of the syntax in the Flux language's source code.
+    The generated response from this endpoint provides a semantic representation
+    of the Flux query with contextual information. The AST breaks down how the
+    query is distributed into different components for execution.
+
+    Use this endpoint for deep query analysis such as debugging unexpected query
+    results.
```

```suggestion
    Analyzes a Flux query and returns a complete package source Abstract Syntax Tree (AST)
    for the  query.
   
    A Flux query AST provides a semantic, tree-like representation with contextual information
    about the query.
    The AST illustrates how the query is distributed into different components for execution.
    
    Use this endpoint for deep query analysis such as debugging unexpected query results.
```

Just a suggestion... always trying to cut words.

---

We should probably add a glossary entry in https://docs.influxdata.com/influxdb/v2.3/reference/glossary and link to it.